naucse/views.py

```diff
@@ -920,3 +936,75 @@ def course_calendar_ics(course):
             abort(404)
 
     return Response(str(calendar), mimetype="text/calendar")
+
+def active_runs():
+    today = datetime.date.today()
+
+    runs = (model.runs_from_year(today.year) +
+            model.runs_from_year(today.year - 1) +
+            model.runs_from_year(today.year - 2))
+    ongoing = [run for run in runs if
+                run.start_date <= today and run.end_date >= today]
+
+    return len(ongoing)
+
+def all_runs():
```

Comment:
Myslím, že je název tento fuknce lehce matoucí, myslím že něco jako `count_all_runs` by bylo lepší, přeci jen to nevrací kurzy, ale jejich počet...